lib/metrics/receivedenvelope.go

```diff
@@ -16,10 +16,20 @@ type ReceivedEnvelope struct {
 }
 
 func (r *ReceivedEnvelope) put(db *sql.DB) error {
-	r.CreatedAt = time.Now().Unix()
-	stmt, err := db.Prepare(`INSERT INTO receivedEnvelopes (messageHash, sentAt, createdAt, pubsubTopic,
-							topic, receiverKeyUID, nodeName, processingError, statusVersion, deviceType)
-							VALUES ($1, $2, $3, $4, $5, $6, $7, $8, $9, $10)
+	tx, err := db.BeginTx(context.Background(), nil)
```

Comment:
Ideally the context should be passed down. Maybe this one could be used https://github.com/status-im/telemetry/blob/master/telemetry/server.go#L39 ?